MyTask/MyTask/Program.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace MyTask
+{
+    internal class Program
+    {
+        static void Main(string[] args)
```

Comment:
Не используемые в коде параметры лучше не писать
```suggestion
        static void Main()
```